core/src/main/java/nu/yona/server/rest/TestController.java

```diff
@@ -145,6 +129,77 @@ private FirebaseMessageResourceAssembler createFirebaseMessageResourceAssembler(
 		return new FirebaseMessageResourceAssembler();
 	}
 
+	/**
+	 * Returns the headers stored in the {@link PassThroughHeadersHolder}. This method blocks till a second request is done, thus
+	 * enforcing multithreading.
+	 * 
+	 * @return the headers stored in the {@link PassThroughHeadersHolder}
+	 */
+	@GetMapping(value = "/passThrougHeaders")
+	@ResponseBody
+	public HttpEntity<PassThroughHeadersResource> getPassThroughHeaders()
+	{
+		Require.that(yonaProperties.isTestServer(),
+				() -> InvalidDataException.onlyAllowedOnTestServers("Endpoint /passThrougHeaders is not available"));
```

Comment:
`passTrougHeaders` -> `passThroughHeaders`

Reply:
You're suffering from a similar issue :)
``passTrougHeaders`` -> ``passThrougHeaders`` -> ``passThroughHeaders``